priv/repo/migrations/20210924034423_create_users.exs

```diff
@@ -0,0 +1,15 @@
+defmodule Twitter.Repo.Migrations.CreateUsers do
+  use Ecto.Migration
+
+  def change do
+    create table(:users) do
+      add :name, :string
+      add :username, :string, null: false
+      add :email, :string, null: false
```

Comment:
you should add a unique constraint for email, too

Reply:
ok